multinet/__init__.py

```diff
@@ -24,7 +24,13 @@
 def create_app(config: Optional[MutableMapping] = None) -> Flask:
     """Create a Multinet app instance."""
     app = Flask(__name__)
-    CORS(app)
+    CORS(
+        app,
+        resources={
+            "/api/.*": {"origins": "*"},
+            "/user/.*": {"origins": "multinet.app", "supports-credentials": True},
```

Comment:
Not saying we should certainly do this, but this would be a good place for the `SERVER_NAME` environment variable. Since this value (`multinet.app`) is dependent on deployment, I think it's best practice to configure this via an environment variable.

Reply:
I agree. That's pretty much how I'm planning to handle it, though I'll use a more precise name than `SERVER_NAME` for the environment variable.